runtime/polkadot/src/lib.rs

```diff
@@ -841,7 +852,7 @@ construct_runtime! {
 		// Vesting. Usable initially, but removed once all vesting is finished.
 		Vesting: vesting::{Module, Call, Storage, Event<T>, Config<T>},
 		// Cunning utilities. Usable initially.
-		Utility: utility::{Module, Call, Storage, Event<T>},
+		Utility: utility::{Module, Call, Event},
```

Comment:
Need to add the new `Multisig` pallet to this runtime.

Reply:
on its way...